components/director/internal/graphql/scalars/annotations.go

```diff
@@ -0,0 +1,32 @@
+package scalars
```

Comment:
Scalars shouldn't be in separate package as they are also GraphQL types. Please move them back to `graphql` package, keeping one file per scalar.

Reply:
done